aspnetcore/tutorials/razor-pages/validation.md

```diff
@@ -57,9 +57,9 @@ The validation attributes specify behavior to enforce on the model properties th
 
 The preceding validation rules are used for demonstration, they are not optimal for a production system. For example, the preceding prevents entering a movie with only two chars and doesn't allow special characters in `Genre`.
 
-Having validation rules automatically enforced by ASP.NET Core helps:
+Having validation rules automatically enforced by ASP.NET Core helpers:
 
-* Helps make the app more robust.
```

Comment:
I think @Rick-Anderson's intention here was to have the verb "helps" on the line that leads into the list and then the predicates are in the bullets. If so, then this just needs the word "Helps" removed from this line and the change on Line 60 can be reverted.

```suggestion
* Make the app more robust.
```